spec/controllers/news_tematica/news_tematicas_controller_spec.rb

```diff
@@ -143,11 +143,11 @@ def post_contenidos_elegidos(prioridades_titulares = nil)
       expect(response).to render_template('news_tematicas/edit')
     end
 
-    it "debe llamar al envío por sendgrid si se ha usado el botón de sendgrid y todo está bien" do
+    it "debe llamar al envío por Mandrill si se ha usado el botón de Mandrill y todo está bien" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.